locales/fil.json

```
@@ -154,60 +154,60 @@
             "wardResidencyRequired": "Ward Residency",
             "website": "Website"
         },
-        "moreInfo": "Click for more info",
+        "moreInfo": "I-click para sa karagdagang impormasyon",
         "report": {
-            "clinicName": "Clinic Name:",
-            "note": "Note about this clinic",
-            "reason": "Why are you reporting this clinic?",
+            "clinicName": "Pangalan ng klinika:",
+            "note": "tungkol sa klinika",
+            "reason": "Bakit mo nirereport ang klinika na ito?",
             "submitReport": "Report",
-            "title": "Report this data"
+            "title": "I-report and detalye"
         },
-        "search": "Search",
+        "search": "Hanapin",
         "voucher": {
-            "notRequired": "Not Required",
-            "required": "Required"
+            "notRequired": "Hindi kailangan",
+            "required": "Kailangan"
         },
         "wardResidency": {
-            "notRequired": "Not Required",
-            "required": "Required"
+            "notRequired": "Hindi kailangan",
+            "required": "Kailangan"
         }
     },
     "delete": {
-        "dialog": "Are you sure you want to delete ID: {0}"
+        "dialog": "Sigurado ka bang gusto mong i-delete ang ID: {0}"
     },
     "donate": {
-        "supportMe": "Support me"
+        "supportMe": "Suportahan mo ako"
     },
     "error": {
         "homePage": "Home page",
-        "notFound": "404 Not Found",
-        "otherError": "An error occurred"
+        "notFound": "404 Hindi Makita",
+        "otherError": "May maling nangyari"
     },
     "findADoc": "Find a Doc",
     "footer": {
-        "donate": "Donate",
+        "donate": "Donasyon",
         "github": "GitHub",
-        "kofi": "Buy me a Ko-Fi",
+        "kofi": "Bilhan mo ako ng kape",
```

```suggestion
        "kofi": "Bilhan mo ako ng Ko-Fi",
```

I think we want to keep "Ko-Fi" here because it's the name of the donation tool. What do you think @bbqbum33 ?

---

Hi Ann, yes let's keep Ko-fi. I confirmed that it still makes sense.